### PR TITLE
Added automation test scripts for TCFv2

### PR DESCRIPTION
**Pull request by deshpandevrushali**

### CODE REVIEW

File: AutomationFramework/src/main/resources/iosDevice.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+[
```
Comment (leodoin) [inline AutomationFramework/src/main/resources/iosDevice.json:1]: @deshpandevrushali , do we need this file for android?

**Comment by leodoin** [comment]
@deshpandevrushali , why are the tests failing?

**Comment by leodoin** [comment]
Do we need all this files that came with the framework, or is there a way to cut it down a bit?

File: AutomationFramework/src/main/resources/iosDevice.json (hunk 1 above)
Comment (deshpandevrushali) [reply AutomationFramework/src/main/resources/iosDevice.json:1]: We are using common framework for both the platforms. This file mention the device details on which we suppose to run our iOS tests and have reference to other files also.
I am planning to work on removing files and references of other platform in second phase once we are done with PR integration.

**Comment by deshpandevrushali** [comment]
I will work on removing files which are related to iOS so that we can have files for Android only.
With last execution one tests failed while loading message. Message not displayed in expected time.

**Comment by deshpandevrushali** [comment]
Removing files related to iOS is planned in second phase.

File: AutomationFramework/src/main/resources/iosDevice.json (hunk 1 above)
Comment (deshpandevrushali) [reply AutomationFramework/src/main/resources/iosDevice.json]: @leodoin I have removed this file and couple of other iOS related files.

**Comment by deshpandevrushali** [comment]
@leodoin Required changes are done. Please have a look so that I can merge the Automation code, Once code is merger TCFv2 Automation scripts will run for further new branches on PR raise.

**Comment by deshpandevrushali** [comment]
@leodoin I have removed 4 files which were related to iOS, you can check this commit
https://github.com/SourcePointUSA/android-cmp-app/pull/177/commits/a6899916a8fadf47f36a42c1dce167e1558df42b

Now we have files related to Android only in source code.

**Pull request merged**